docs/src/main/asciidoc/Bootable_Guide.adoc

```diff
@@ -0,0 +1,197 @@
+[[Bootable_Guide]]
+= Bootable JAR Guide
+WildFly team;
+:revnumber: {version}
+:revdate: {localdate}
+:toc: macro
+:toclevels: 3
+:toc-title: Bootable JAR Guide
+:doctype: book
+:icons: font
+:source-highlighter: coderay
+ifdef::env-github[:imagesdir: images/]
+
+// ifndef::ebook-format[:leveloffset: 1]
+
+(C) 2020 The original authors.
+
+ifdef::basebackend-html[toc::[]]
+:numbered:
+
+[[wildfly_bootable_JAR_development]]
+= WildFly bootable JAR application development
+
+This document details the steps to follow in order to develop a WildFly application
+packaged as a bootable JAR. A bootable JAR can be run both on bare-metal and cloud platforms.
+
+This document contains the minimal information set required to build and run a WildFly bootable JAR. 
+Complete information on the Maven plugin usage can be found in the bootable JAR https://docs.wildfly.org/bootablejar/[documentation].
+
+Developing an application packaged as a bootable JAR is not different from developing an application for WildFly server.
+The extra steps required to package your application inside a bootable JAR are handled by the 
+https://github.com/wildfly-extras/wildfly-jar-maven-plugin[_org.wildfly.plugins:wildfly-jar-maven-plugin_] Maven plugin.
+
+
+[[wildfly-jar-maven-plugin-pom-configuration]]
+== Adding the bootable JAR Maven plugin to your pom file
+
+Add the plugin to your Maven build:
+
+[source,xml]
+----
+<build>
+  <plugins>
+    <plugin>
+      <groupId>org.wildfly.plugins</groupId>
+      <artifactId>wildfly-jar-maven-plugin</artifactId>
```

Comment:
It is starting to feel a bit odd that we have a discrepancy between how we name the plugin and how we name the plugin.

i.e. we have chosen to give the plugin the artifact ID "wildfly-jar-maven-plugin" but we keep referring to it as "bootable jar plugin"

Reply:
The artifact has a shorter name, jar implies bootable. I think we should use bootable for some time.